charts/testkube-logs/values.yaml

```diff
@@ -174,7 +174,7 @@ storage:
   # -- MinIO Token
   token: ""
   # -- MinIO Bucket
-  bucket: "testkube-artifacts"
+  bucket: "testkube-new-logs"
```

Comment:
we discussed it @exu and guess we need to use the same bucket for logs like in v1